crates/aptos-logger/Cargo.toml

```diff
@@ -15,6 +15,7 @@ rust-version = { workspace = true }
 # Do NOT add any inter-project dependencies.
 # This is to avoid ever having a circular dependency with the aptos-logger crate.
 [dependencies]
+aptos-build-info = { workspace = true }
```

Comment:
We cannot depend on this crate here and need to pass the git hash info from aptos-node. There is an age old problem where if we included it as part of aptos-logger it would make cargo build very very slow. That was one of the initial reason this crate was created.